util/Infix.re

```diff
@@ -34,7 +34,7 @@ let logIfAbsent = (message, x) => switch x {
 };
 
 let maybeConcat = (a, b) => {
-  if (b != "" && b.[0] == '/') {
+  if (b != "" && (b.[0] == '/' || b.[1] == ':')) {
```

Comment:
TODO: Need to make this more robust - big problems if we have a single character path 😉

Reply:
Addressed in https://github.com/jaredly/reason-language-server/pull/199/commits/6e017420c70358dbff98d7fe8a0b07da436d2511